Pear/Controller/CommunityViewController.swift

```diff
@@ -113,6 +117,11 @@ class CommunityViewController: UIViewController {
         searchBar.resignFirstResponder()
     }
     
+    @objc func refreshSearches() {
+        getUsers()
+        searchRefreshControl.endRefreshing()
```

Comment:
We wouldn't want to end refreshing here right? We'd want to end refreshing after our network call is completed (with either a failure or success)